packages/commonwealth/client/scripts/controllers/server/rules.ts

```diff
@@ -0,0 +1,154 @@
+import RuleStore from 'stores/RulesStore';
+import app from 'state';
+import $ from 'jquery';
+import { Rule } from 'models';
+
+export const ruleModelFromServer = (ruleFromDb) => {
```

Comment:
Don't like this style -- we should use `fromJSON` on the `Rule` model itself + ensure types.